.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,16 @@
+name: C++ Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

Comment:
Wana freeze this to something like 20.04 (latest LTS?)

Reply:
Yep, I did that in 242990419ed3f3607aa9c7246f99efa2cd303c36